lib/index.js

```diff
@@ -817,7 +817,14 @@ SQLVarText.prototype.decode = function(data) {
     }
 
     if (!data.readInt()) {
+		/***********************************
```

Comment:
If I had the chance to review before this pr was merged, I would have warned about this weird indentation. This will be probably be refused on the main node-firebird project.